src/components/Crop.tsx

```diff
@@ -134,11 +134,10 @@ const FourDivs = (
 );
 
 const cropStyle = (box: CropperBox, style: CSSProperties): CSSProperties => {
-  const { x, y, width, height } = box;
-
+  const { x, y, width, height, originX ,originY  } = box;
   return {
     ...style,
-    transformOrigin: `${-box.x}px ${-box.y}px`,
+    transformOrigin: `${-originX || x}px ${-originY || y}px`,
```

Comment:
It'll read `x` or `y` when originX/Y are 0. Is that intended? Or did you mean to check against null/undefined?

If only null or undefined, consider using the bullish coalescing operator like `-originX ?? x`.

Reply:
i was checking against null/undefined,  no need of originX || x, we can just use originX, let me fix this and commit

Reply:
commited, ran format as well